Keas.Mvc/Services/ServiceNowService.cs

```diff
@@ -27,9 +27,15 @@ public ServiceNowService(IOptions<ServiceNowSettings> serviceNowSettings)
 
         public async Task<ServiceNowPropertyWrapper> GetComputersByName(string name)
```

Comment:
You should change this to be something like `GetComputersByProperty(string property)` since it isn't just name anymore